### PR TITLE
An example for 3rd operator upgrade automation

### PR DESCRIPTION
**Pull request by jianzhangbjz**
@anpingli @chengzhang1016 @emmajiafan @cuipinghuo Please have a review, thanks!

### CODE REVIEW

File: features/upgrade/tsb/upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    And the output should contain "No resources found"
+    """
+    # Recreate a TSB
+    When I process and create:
+      | f | https://raw.githubusercontent.com/jianzhangbjz/v3-testfiles/tsb-upgrade/olm/asb-cr-template.yaml |
+    Then the step should succeed
+    And I wait up to 180 seconds for the steps to pass:
+    """
+    And a pod becomes ready with labels:
+        | app=template-service-broker |
```
Comment (pruan-rht) [inline features/upgrade/tsb/upgrade.feature]: indentation issue.

File: features/step_definitions/operators.rb [hunk 2]
```diff
@@ -49,3 +49,9 @@
   })
   step %Q/the step should succeed/
 end
+
+# Get the Major.Minor cluster version
+Given /^the major.minor version of the cluster is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
+  cb_name = 'operator_channel_name' unless cb_name
+  cb[cb_name] = cluster_version('version').channel.split('stable-')[1]
```
Comment (pruan-rht) [inline features/step_definitions/operators.rb]: @jianzhangbjz are you sure `stable-` is always there?

File: features/upgrade/tsb/upgrade.feature (hunk 1 above)
Comment (jianzhangbjz) [reply features/upgrade/tsb/upgrade.feature]: Thanks!

File: features/step_definitions/operators.rb (hunk 2 above)
Comment (jianzhangbjz) [reply features/step_definitions/operators.rb]: @pruan-rht Thanks! I guess `-` always there. Changed it to `-`.

**Comment by jianzhangbjz** [comment]
Merge it so that others can use these basic steps. Thanks!

**Pull request merged**